pkg/render/logstorage.go

```diff
@@ -522,22 +526,32 @@ func (es elasticsearchComponent) elasticsearchCluster() *esv1.Elasticsearch {
 			NodeSets: []esv1.NodeSet{
 				{
 					Count: int32(nodeConfig.Count),
-					Name:  "es",
+					Name: "es-" + nodeSetNameSuffix(pvcTemplate),
```

Comment:
You can omit the "es-" bit

Reply:
Is a prefix added to the node set automatically? If not we should add "tigera" or something like that.

Reply:
no, what we give is used as it is for NodeSet name, with the current change pod name becomes something like `tigera-secure-es-es-1dacabc7d15654312c8c-0` from original `tigera-secure-es-es-0`.

Reply:
I'll remove `es-` prefix from the NodeSet name, it'll be less redundant, so the statefulset generated will be `tigera-secure-es-<thumbprint>`

Reply:
Is it hard to get a shorter hash?

Reply:
> You can omit the "es-" bit
> Is a prefix added to the node set automatically? If not we should add "tigera" or something like that.

Agree with Rene. You don't need any prefix in the NodeSet name, because ECK already adds another prefix (from the Elasticsearch resource name). 

I was curious about this myself, so I looked at the Elastic docs. 

[Doc states](https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-orchestration.html#k8s-statefulsets): 
> the StatefulSet name is built from the Elasticsearch resource name and the NodeSet name. Each Pod will be assigned the StatefulSet name suffixed by an ordinal. The corresponding Elasticsearch node has the same name as the Pod.

So for example, if the NodeSet name is `es` and our Elasticsearch resource name is `tigera-secure` (which is what it is currently), then from the Elastic docs it sounds like StatefulSet name should be `tigera-secure-es`. It's actually `tigera-secure-es-es` (I checked in a running cluster). 

The Elasticsearch pod created from the StatefulSet is called `tigera-secure-es-es-0`. 

So other than the double `es` it seems to fit what the documentation says.

Reply:
>the StatefulSet name is built from the Elasticsearch resource name and the NodeSet name. Each Pod will be assigned the StatefulSet name suffixed by an ordinal. The corresponding Elasticsearch node has the same name as the Pod.
https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-orchestration.html#k8s-statefulsets
Thats how we are getting `tigera-secure-es-es-0`